routers/web/repo/editor.go

```diff
@@ -69,6 +69,9 @@ func editFile(ctx *context.Context, isNewFile bool) {
 	ctx.Data["IsNewFile"] = isNewFile
 	ctx.Data["RequireHighlightJS"] = true
 	ctx.Data["RequireSimpleMDE"] = true
+	if len(ctx.Req.URL.Query().Get("value")) > 0 {
+		ctx.Data["FileContent"] = ctx.Req.URL.Query().Get("value")
```

Comment:
```suggestion
	if len(ctx.Query("value")) > 0 {
		ctx.Data["FileContent"] = ctx.Query("value")
```

Reply:
We should only get parameters from url but not the body. The current `Query` in fact will also try to parse the body. So I agree with @coolaj86